tests/lib/mod.rs

```diff
@@ -23,7 +24,11 @@ fn schema_from_file(file: &str) -> PathBuf {
 fn teo_exe_path_buf() -> PathBuf {
     let mut current_dir = env::current_dir().unwrap();
     while current_dir != PathBuf::from("/") {
-        let exe_path = current_dir.join("target/debug/cargo-teo");
+        let exe_path = if whoami::platform() == Platform::Windows {
```

Comment:
Hi @l1xnan, let's move this condition into the join argument

Reply:
Let me modify that.